common/djangoapps/custom_settings/helpers.py

```diff
@@ -1,29 +1,40 @@
-import pytz
+"""
+All helpers for custom settings app
+"""
 from datetime import datetime
+
+import pytz
+
 from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
 
 DATE_FORMAT = "%m/%d/%Y"
 
 
 def get_course_open_date_from_settings(settings):
     """
-    :param settings:
-    :return course open date:
+    Get course open date from settings, if exists, in MM/DD/YYY format else return empty string
+
+    Args:
+        settings (CustomSettings): Custom settings model object
+
+    Returns:
+        string: Date in MM/DD/YYY format or empty string
```

Comment:
Can we change this format detail to  %m/%d/%Y ? it looks confusing to me.

Reply:
you're right %m/%d/%Y looks developer specific, Please change it to "MM/DD/YYYY" as we discussed. Thank you :)